ui/component/viewers/videoViewer/internal/plugins/videojs-overlay/plugin.scss

```diff
@@ -81,5 +81,163 @@
   .vjs-no-flex .vjs-overlay-right {
     margin-top: $offset-v;
   }
+}*/
+
+$accent-color: var(--color-primary);
```

Comment:
Please just use `var(--color-primary)` everywhere, or add a new line to the light/dark.scss called something like `--video-accent-color` (even if that is still `--color-primary` under the hood